app/models/permision.rb

```diff
@@ -0,0 +1,8 @@
+class Permision < ApplicationRecord
+  belongs_to :users
+  belongs_to :room
+
+  validates :user_id, uniqueness: { scope: :room_id }
```

Comment:
what if you are `admin` but you are muted ( only reader ) ?

Reply:
I don't understand what you mean

I think admin can't be muted . All of admin has same permission (admin can't mute other admin)